src/Helpers/media_library_helpers.php

```diff
@@ -63,9 +63,6 @@ function bytesToHuman($bytes)
      */
     function replaceAccents($str)
     {
-        if (function_exists('mb_convert_encoding')) {
-            return mb_convert_encoding($str, 'ASCII', 'UTF-8');
```

Comment:
I had to remove this after adding tests `é` would be converted into `?` instead of `e` which could result in empty filenames